.github/workflows/ci_push.yml

```diff
@@ -29,10 +29,20 @@ jobs:
         .github/workflows/install_deps.sh amici
         pip install .
 
+    - name: flake8
+      run: |
+        chmod u+x ./run_flake8.sh
```

Comment:
this only needs to be done once on your system locally, as usage rights are also "gitted"